Dockerfile

```
@@ -0,0 +1,17 @@
+FROM node:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)